### PR TITLE
Add Whitelist

### PR DESCRIPTION
**Pull request by dave-tucker**
The initial whitelist was compiled by @riyazdf
This is required to check that a PR is from an approved author before
running it through the CI tests. This allows us to run PR's against a
wide range of cloud providers without worrying about our secrets being
compromised.

Required by #10

Signed-off-by: Dave Tucker <dt@docker.com>

### CODE REVIEW

**Review by riyazdf — APPROVED** [review]
LGTM

cc @justincormack @rn @ijc

File: whitelist [hunk 1]
```diff
@@ -0,0 +1,30 @@
+justincormack
+rneugeba
+ijc25
+riyazdf
+djs55
+dsheets
+samoht
+nathanleclaire
```
Comment (rn) [inline whitelist]: Nathan is not working on LinuxKit anymore

File: whitelist [hunk 2]
```diff
@@ -0,0 +1,30 @@
+justincormack
+rneugeba
```
Comment (rn) [inline whitelist]: my github ID changed to `rn`

File: whitelist [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ijc25
```
Comment (rn) [inline whitelist]: `ijc25` is not known as `ijc`

**Review by rn — COMMENTED** [review]
a few comments

**Comment by dave-tucker** [comment]
Thanks for the reviews. Pushed an update with @rn's comments addressed.

**Review by rn — APPROVED** [review]

**Review by justincormack — APPROVED** [review]

**Pull request merged**